Debugger error when trying to debug tests

**Describe the bug**
There's a debugger error (`Error: Cannot find module '/Users/hypest/proj/a8c/rn/tmp/t27/gutenberg-mobile/jest'`) happening after attaching debugger to the tests run.

**To Reproduce**
Steps to reproduce the behavior:
1. Run `yarn test:debug` on the CLI
2. Open Chrome, and visit: `chrome://inspect` and hit "Inspect" on the `jest file:///` target
3. Notice the debugger immediately errorring with `Error: Cannot find module '/Users/hypest/proj/a8c/rn/tmp/t27/gutenberg-mobile/jest'` 💥 

**Expected behavior**
Debugger should continue and only stop on runtime errors/breakpoints.

**Smartphone (please complete the following information):**
Note: the error is happening on a macos v10.14.6 laptop.

 - Version v1.16.0, hash: `1995c3ae`

--- a/package.json
+++ b/package.json
@@ -90,7 +90,7 @@
     "ios": "react-native run-ios",
     "test": "cross-env NODE_ENV=test jest --verbose --config jest.config.js",
     "test:inside-gb": "cross-env NODE_ENV=test jest --verbose --config jest_gb.config.js",
-    "test:debug": "cross-env NODE_ENV=test node --inspect-brk jest --runInBand --verbose --config jest.config.js",
+    "test:debug": "cross-env NODE_ENV=test node --inspect-brk node_modules/.bin/jest --runInBand --verbose --config jest.config.js",
     "device-tests": "cross-env NODE_ENV=test jest --no-cache --maxWorkers=3 --reporters=default --reporters=jest-junit --verbose --config jest_ui.config.js",
     "device-tests:local": "cross-env NODE_ENV=test jest --runInBand --reporters=default --reporters=jest-junit --detectOpenHandles --verbose --config jest_ui.config.js",
     "device-tests:debug": "cross-env NODE_ENV=test node $NODE_DEBUG_OPTION --inspect-brk node_modules/jest/bin/jest --runInBand --reporters=default --reporters=jest-junit --detectOpenHandles --verbose --config jest_ui.config.js",

